frontend/actions/review.ts

```diff
@@ -34,11 +34,11 @@ type SemverOrAccessRequestId =
     }
   | {
       semver?: never
-      accessRequestId: AccessRequestInterface['id']
+      accessRequestId: AccessRequestInterface['id'] | undefined
```

Comment:
I came across this same problem with releases. If we make these optional, we lose the guidance/safety that this type provides.

Instead we can do the following:
```
type SemverOrAccessRequestId =
  | {
      semver: ReleaseInterface['semver'] | ''
      accessRequestId?: never
    }
  | {
      semver?: never
      accessRequestId: AccessRequestInterface['id'] | ''
    }
```
That way we can use it like so (in your `[accessRequestId].tsx` component) and still keep the either/or behaviour for semver/requestId:
```
  const {
    reviews: activeReviews,
    isReviewsLoading: isActiveReviewsLoading,
    isReviewsError: isActiveReviewsError,
  } = useGetReviewRequestsForModel({
    modelId: modelId,
    accessRequestId: accessRequestId || '',
    isActive: true,
  })
```

If we tried to provide a `semver` along with the above, we'll get the error saying that it's invalid. Which is what we want :)